### PR TITLE
feat(preview): add responsive behavior to preview drawer

### PR DESCRIPTION
**Pull request by greathmaster**
Summary of changes:

for screen sizes less than 768px...
- moved sidebar drawer to bottom 
- animated sidebar drawer open and close
- shows greyscale version of Box Sign icon
- hides the additional apps section

Demo
Responsive-Preview-Drawer

### CODE REVIEW

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
```
Comment (ConradJChan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:17]: Should these be applied to the base style above instead of only for the medium breakpoint?

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
```
Comment (ConradJChan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:22]: Does this `static` position affect the way annotations are rendered on top of preview?

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
+        }
+
+        .preview-body {
+            height: 100%;
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:32]: Relative height is notoriously difficult to work with, since it has to either have a fixed-height parent or go all the way up to the root of the document. Should we use flexbox or absolute positioning, instead?

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        }
+
+        .be {
+            height: 100%;
+
+            .bcs-activity-feed {
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:38]: Can these styles be moved to the components they affect?

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+        }
+
+        .be.bcs {
+            flex-direction: column;
+            width: 100%;
+            min-width: unset;
+            max-width: unset;
+            height: 100%;
+            max-height: 48px;
+            -webkit-transition: max-height .5s ease-in-out 0s;
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:54]: We should have autoprefixer, making this unnecessary.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+            .bcs-scroll-content {
+                width: unset;
+            }
+        }
+
+        .be.bcs.bcs-is-open {
+            position: absolute;
+            bottom: 0;
+            z-index: 2;
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:70]: Is this needed? If so, does it work properly with all other floating/tethered elements (annotations, tooltips, modals, etc.)?

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+        .bdl-AdditionalTabs > *:not(.bdl-MoreOptions) {
+            display: none;
+        }
+
+        .bcs .bcs-SidebarNav-overflow .bdl-AdditionalTabs {
+            padding: 0;
+        }
+
+        .bcs .bcs-SidebarNav-overflow::after,
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:112]: Let's revisit the amount of specificity we're using and also consolidate our style rules for the same components using SCSS nesting.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+        .bcs .bcs-SidebarNav-overflow::before {
+            display: none;
+        }
+
+        .bcs .bcs-SidebarNav-footer {
+            flex: 1 1 48px;
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarResponsiveness.scss:118]: What is the source of this value? Should it be a variable?

File: .storybook/preview-body.html [hunk 9]
```diff
@@ -1,6 +1,7 @@
 <style>
     #root {
         padding: 16px;
+        position: relative;
```
Comment (jstoffan) [inline .storybook/preview-body.html:4]: Is this needed? Will customers using Preview need to add this to their stylesheets for the new styles to work?

File: src/elements/content-sidebar/SidebarNavSign.tsx [hunk 10]
```diff
@@ -55,6 +56,7 @@ export function SidebarNavSign({ blockedReason, intl, status, targetingApi, ...r
             <Tooltip isDisabled={isTargeted} position={TooltipPosition.MIDDLE_LEFT} text={tooltipMessage}>
                 <PlainButton aria-label={label} className={buttonClassName} isDisabled={isSignDisabled} {...rest}>
                     <BoxSign28 className="bcs-SidebarNavSign-icon" />
+                    <Sign16 className="u-md-bcs-SidebarNavSign-icon" />
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNavSign.tsx:59]: This was requested by Product to change the Box Sign icon to the grayscale version when it shifts to the horizontal drawer on small screens.

File: src/elements/content-sidebar/additional-tabs/AdditionalTab.js [hunk 11]
```diff
@@ -101,6 +101,7 @@ class AdditionalTab extends React.PureComponent<Props, State> {
         const className = classNames('bdl-AdditionalTab', {
             'bdl-is-hidden': isLoading,
             'bdl-is-disabled': isDisabled,
+            'bdl-MoreOptions': id && id < 0,
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/additional-tabs/AdditionalTab.js:104]: Done.

[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/additional-tabs/AdditionalTab.js:104]: Going to go with @jstoffan suggestion of `bdl-is-overflow`

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 1 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:17]: The greyscale Sign icon is only meant to show up on medium screen sizes and below, so it's going to be hidden on anything bigger. Functionally, it would end up being the same. I'll go ahead and update it if it's more clear that way.

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 2 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:22]: Annotations are unaffected by the `position: static`:
Screen Shot 2022-06-29 at 1 44 10 PM

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 5 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:54]: Understood. Going to remove vendor prefixes.

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 6 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:70]: The `z-index` is needed so the drawer displays over the Preview Header when expanded.

Screen Shot 2022-06-29 at 3 12 30 PM

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+                width: unset;
+            }
+        }
+
+        .be.bcs.bcs-is-open {
+            position: absolute;
+            bottom: 0;
+            z-index: 2;
+            height: 100%;
+            max-height: 100%;
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:72]: The `max-height` is being used as the transition property for the drawer animation. Using it just on `height` would cause the drawer to animate when resized.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+
+        .bcs .bcs-NavButton,
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 48px;
+            height: 48px;
+            margin-right: 12px;
+        }
+
+        .bdl-AdditionalTabs > *:not(.bdl-MoreOptions) {
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:104]: Updated to use `.bdl-AdditionalTab` instead.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 14]
```diff
@@ -0,0 +1,154 @@
...
+            min-width: unset;
+            max-width: unset;
+            height: 100%;
+            max-height: 48px;
+            -webkit-transition: max-height .5s ease-in-out 0s;
+            transition: max-height .5s ease-in-out 0s;
+        }
+
+        .bcs-content {
+            width: unset;
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:59]: Yes. Also updated `auto` in couple other places as well.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 15]
```diff
@@ -0,0 +1,154 @@
...
+            position: absolute;
+            bottom: 0;
+            z-index: 2;
+            height: 100%;
+            max-height: 100%;
+            -webkit-transition: max-height .5s ease-in-out 0s;
+            transition: max-height .5s ease-in-out 0s;
+        }
+
+        .bcs {
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:77]: Removed the nesting.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 16]
```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:3]: Seems I misunderstood how the `u-` prefix is used. I've updated the classname to `bcs-SidebarNavSign-icon--greyscale` instead.

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 3 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:32]: Instead of using `height: 100%` I've updated it to use `flex: basis: 100%` instead.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 17]
```diff
@@ -0,0 +1,154 @@
...
+            .bcpr-container {
+                position: static;
+            }
+        }
+
+        .preview-body {
+            height: 100%;
+        }
+
+        .be {
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:35]: Removed the `height: 100%` for the `.be` class as it was unnecessary.

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 4 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:38]: Moved to `ActivityFeed.scss`

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 7 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:112]: Consolidated styles and reduced specificity in many places.

File: src/elements/content-sidebar/ContentSidebar.js [hunk 18]
```diff
@@ -39,6 +39,7 @@ import '../common/fonts.scss';
 import '../common/base.scss';
 import '../common/modal.scss';
 import './ContentSidebar.scss';
+import './SidebarResponsiveness.scss';
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/ContentSidebar.js:42]: Managed to move all the styles into their appropriate files so deleting `SidebarResponsive.scss`.

File: .storybook/preview-body.html (hunk 9 above)
Comment (greathmaster) [reply .storybook/preview-body.html:4]: Adding `position: relative` here is to anchor the `position: absolute` used further down.

I'm unsure if customers will need to add this to their stylesheets.

For that reason, I've updated the code to remove it from `#root` and instead add `position: relative` to the `.be.bcpr` selector. This will leave the `preview-body.html` file unchanged.

This looks okay in Storybook (image below). However, in EUA, we add an additional header with the filename (image below).

To address this I will need to submit a PR to EUA that unsets the `be.bcpr` selector and instead adds the `position: relative` to the `.preview-container`. Seems like a better solution than adding `position: relative` to the `#root` element.

**Storybook**
Screen Shot 2022-07-01 at 6 26 15 PM

**EndUserApp would need a couple additional styles.**
Screen Shot 2022-07-01 at 6 08 06 PM

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 19]
```diff
@@ -0,0 +1,154 @@
...
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:15]: I went ahead and did this, but could I get some clarification on why this is better than specifying in CSS?

Seems setting it in JSX makes it hard to change or override the style later?

File: src/elements/content-sidebar/SidebarResponsiveness.scss (hunk 8 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:118]: Did some more testing and seems we don't need this property. I've removed it.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 20]
```diff
@@ -0,0 +1,154 @@
...
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:27]: I've tested this across several formats and it seems to be okay. Additionally, from what I have tested, Annotations render correctly.

File: src/elements/content-sidebar/SidebarResponsiveness.scss [hunk 21]
```diff
@@ -0,0 +1,154 @@
...
+            }
+        }
+
+        .preview-body {
```
[the remark this answers is not in the document]
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarResponsiveness.scss:31]: Personally, I don't think we should move the styles to the Preview SDK because, the additional styles are in support of the Sidebar moving to the bottom on small screens. My thinking is it's better to keep all the functionality in one repo rather than splitting it.

File: src/elements/content-sidebar/SidebarNav.scss [hunk 22]
```diff
@@ -63,3 +63,88 @@
...
+        .bcs-NavButton::before {
+            right: 0;
+            bottom: 0;
+            left: 0;
+            height: 2px;
+        }
+
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 48px;
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:97]: This seems to be a common measurement. Can it be moved to a local variable?

File: src/elements/content-sidebar/SidebarNav.scss [hunk 23]
```diff
@@ -63,3 +63,88 @@
...
+        .bcs-SidebarNav-tabs {
+            flex-direction: row;
+        }
+
+        .bcs-SidebarNav-main {
+            display: flex;
+            flex-direction: row;
+        }
+
+        .bcs-NavButton::before {
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:88]: Can we group this with and move after the other `.bcs-NavButton` styles? Can we also move those styles to the SidebarNavButton component stylesheet?

File: src/elements/content-sidebar/SidebarNav.scss [hunk 24]
```diff
@@ -63,3 +63,88 @@
...
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 46px;
+            margin-right: 0;
+        }
+    }
+
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNav,
+        .bcs-SidebarNav-tabs {
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:79]: Can we add `.bcs-SidebarNav-main` to the list of classes here?

File: src/elements/content-sidebar/SidebarNav.scss [hunk 25]
```diff
@@ -63,3 +63,88 @@
         }
     }
 }
+
+.is-responsive-web {
+    @include breakpoint($small-screen) {
+        .bcs .bcs-NavButton,
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 46px;
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:72]: Why did we lose a couple pixels here?

File: src/elements/content-sidebar/SidebarNav.scss [hunk 26]
```diff
@@ -63,3 +63,88 @@
...
+            }
+
+            &::after,
+            &::before {
+                display: none;
+            }
+        }
+
+        .bcs-NavButton {
+            width: 48px;
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:124]: Duplicated styles from above. Should they be moved to a mixin?

File: src/elements/content-sidebar/SidebarNav.scss [hunk 27]
```diff
@@ -63,3 +63,88 @@
...
+        }
+
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 48px;
+            height: 48px;
+            margin-right: 12px;
+        }
+
+        .bdl-AdditionalTabs > .bdl-AdditionalTab:not(.bdl-is-overflow),
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:102]: Is the extra specificity of a direct descendent selector needed here? We don't nest these components.

File: src/elements/content-sidebar/SidebarNav.scss [hunk 28]
```diff
@@ -63,3 +63,88 @@
...
+        .bdl-AdditionalTabs > .bdl-AdditionalTabPlaceholder {
+            display: none;
+        }
+
+        .bdl-AdditionalTabs > :last-child,
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:107]: What are we selecting here?

File: src/elements/content-sidebar/SidebarNav.scss [hunk 29]
```diff
@@ -63,3 +63,88 @@
...
+            margin-right: 12px;
+
+            &::before {
+                top: 48px;
+                width: auto;
+            }
+        }
+
+        .bcs-SidebarNav-footer {
+            .btn-plain.bdl-SidebarToggleButton {
```
Comment (jstoffan) [inline src/elements/content-sidebar/SidebarNav.scss:135]: Can these styles be moved to the SidebarToggleButton stylesheet?

File: src/elements/content-sidebar/ContentSidebar.scss [hunk 30]
```diff
@@ -28,3 +28,26 @@ $sidebarWidth: $sidebarTabsWidth + $sidebarContentWidth;
...
+            max-height: 48px;
+            transition: max-height .5s ease-in-out 0s;
+
+            &.bcs-is-open {
+                position: absolute;
+                bottom: 0;
+                z-index: 2;
+                flex-basis: 100%;
+                max-height: 100%;
+                transition: max-height .5s ease-in-out 0s;
```
Comment (jstoffan) [inline src/elements/content-sidebar/ContentSidebar.scss:49]: Do we need to re-declare the transition here?

File: src/elements/content-sidebar/SidebarNav.scss (hunk 25 above)
Comment (jstoffan) [reply src/elements/content-sidebar/SidebarNav.scss:72]: It looks like I asked this and it was answered [previously](https://github.com/box/box-ui-elements/pull/2917#discussion_r907874219). Can we add a brief comment in the code to avoid confusion in the future?

File: src/elements/content-sidebar/SidebarNavSign.tsx [hunk 31]
```diff
@@ -55,6 +56,7 @@ export function SidebarNavSign({ blockedReason, intl, status, targetingApi, ...r
             <Tooltip isDisabled={isTargeted} position={TooltipPosition.MIDDLE_LEFT} text={tooltipMessage}>
                 <PlainButton aria-label={label} className={buttonClassName} isDisabled={isSignDisabled} {...rest}>
                     <BoxSign28 className="bcs-SidebarNavSign-icon" />
+                    <Sign16 width={20} height={20} className="bcs-SidebarNavSign-icon--greyscale" />
```
Comment (tjuanitas) [inline src/elements/content-sidebar/SidebarNavSign.tsx:59]: nit: majority of spelling in this repo is `gray` vs `grey`

```suggestion
                    <Sign16 width={20} height={20} className="bcs-SidebarNavSign-icon--grayscale" />
```

File: src/elements/content-sidebar/ContentSidebar.scss [hunk 32]
```diff
@@ -28,3 +28,26 @@ $sidebarWidth: $sidebarTabsWidth + $sidebarContentWidth;
...
+            min-width: 0;
+            max-width: none;
+            max-height: 48px;
+            transition: max-height .5s ease-in-out 0s;
+
+            &.bcs-is-open {
+                position: absolute;
+                bottom: 0;
+                z-index: 2;
+                flex-basis: 100%;
```
Comment (tjuanitas) [inline src/elements/content-sidebar/ContentSidebar.scss:47]: Is setting `flex-basis` needed here since it's being set above?

File: src/elements/content-sidebar/SidebarNav.scss (hunk 22 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNav.scss:97]: Moved it to a local variable `$sidebarTabResponsiveSize` and updated properties.

File: src/elements/content-sidebar/SidebarNav.scss (hunk 23 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNav.scss:88]: Consolidated `.bcs-NavButton` styles, and moved it to `SidebarNavButton.scss`

File: src/elements/content-sidebar/SidebarNav.scss (hunk 25 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNav.scss:72]: Added a comment to clarify.

File: src/elements/content-sidebar/SidebarNav.scss (hunk 27 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNav.scss:102]: Tested this...seems like it isn't. Reduced specificity.

File: src/elements/content-sidebar/SidebarNav.scss (hunk 28 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNav.scss:107]: The last child in the `bdl-AdditionalTabs` is the ... button for More Apps. While the last child in `bcs-SidebarNav-secondary` is the Sign button. Setting the `margin-right: 0` for last child of both of these avoids pushing out the footer button:

#### Without setting `margin: 0`
Screen Shot 2022-07-13 at 5 13 17 PM

File: src/elements/content-sidebar/SidebarNav.scss (hunk 29 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNav.scss:135]: Maybe...The regular (non-responsive) `.bcs-SidebarNav-footer` style is already in `SidebarNav.scss`:

https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/SidebarNav.scss#L52-L64

So I think we should either move both of them or keep both in `SidebarNav.scss`.

File: src/elements/content-sidebar/ContentSidebar.scss (hunk 30 above)
Comment (greathmaster) [reply src/elements/content-sidebar/ContentSidebar.scss:49]: Hmm...actually we don't.

File: src/elements/content-sidebar/SidebarNavSign.tsx (hunk 31 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNavSign.tsx:59]: Got it. Going with `gray`

File: src/elements/content-sidebar/ContentSidebar.scss (hunk 32 above)
Comment (greathmaster) [reply src/elements/content-sidebar/ContentSidebar.scss:47]: Yeah. You're right. We don't need it again. Removed.

File: src/elements/content-sidebar/SidebarNav.scss (hunk 26 above)
Comment (greathmaster) [reply src/elements/content-sidebar/SidebarNav.scss:124]: Not sure this is still applicable, since it was moved to `SideNavButton.scss`?

File: src/elements/content-preview/ContentPreview.scss [hunk 33]
```diff
@@ -75,3 +75,22 @@
...
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
```
Comment (tjuanitas) [inline src/elements/content-preview/ContentPreview.scss:93]: question: Is this for modifying the position of the navigation buttons (`.bcpr-navigate-left` and `.bcpr-navigate-right`) to make them vertically centered with the screen as opposed to the preview container when the sidebar is on the bottom?

If not, what's the reason for this change?

File: src/elements/content-preview/ContentPreview.scss [hunk 34]
```diff
@@ -75,3 +75,22 @@
...
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .be.bcpr {
+            position: relative;
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
```
Comment (tjuanitas) [inline src/elements/content-preview/ContentPreview.scss:87]: Is `position: static` needed?

For screen sizes between 700px (`.be-is-small`) and 768px, the position is `static` by default.

For screen sizes 700px and less (i.e. when `.be-is-small` is applied), `.bcpr-body` has `position: relative;` ([from this](https://github.com/box/box-ui-elements/blob/master/src/elements/content-preview/ContentPreview.scss#L63-L66)) but because `.bcpr` and `.bcpr-body` are the same sizes, this shouldn't have an effect.

Comment (tjuanitas) [reply src/elements/content-preview/ContentPreview.scss:87]: Alternatively, I think we could set `.bcpr-body` to `position: relative;` here and then remove these lines: https://github.com/box/box-ui-elements/blob/3b8e7767d3b88ec926e99c278f37843c1292ade1/src/elements/content-preview/ContentPreview.scss#L81-L83

This allows us in the future to remove this: https://github.com/box/box-ui-elements/blob/3b8e7767d3b88ec926e99c278f37843c1292ade1/src/elements/content-preview/ContentPreview.scss#L64-L66

Comment (greathmaster) [reply src/elements/content-preview/ContentPreview.scss:87]: The main reason for applying `position: static` was to ensure the `position: absolute` from `aside` had a clear path to anchor on the `position relative` of 

`.be.bcpr { 
     position: relative; 
 } `

I tried the above and in Storybook it seems the Preview drawer doesn't fully cover the filename header when extended.

localhost_6061__path=_story_elements-contentpreview--preview-with-annotations

**Review by tjuanitas — APPROVED** [review]

**Comment by tjuanitas** [comment]
@Mergifyio unqueue

**Comment by tjuanitas** [comment]
@Mergifyio refresh
